### ISSUE TITLE
Editor complaining about crashes on PyCharm 2021.1

### ISSUE THREAD

**Issue by Mike3285** [SeeSharpSoft/intellij-csv-validator#274]
Hello, the plugin is crashing on my editor, showing an error message.

My editor:

```
PyCharm 2021.1 (Professional Edition)
Build #PY-211.6693.115, built on April 6, 2021
Runtime version: 11.0.10+9-b1341.35 amd64
VM: Dynamic Code Evolution 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 1960M
Cores: 12
Registry: documentation.show.toolbar=true
Non-Bundled Plugins: de.docs_as_co.intellij.plugin.diagramsnet (0.1.5), ideanginx9 (0.1.9), net.seesharpsoft.intellij.plugins.csv (2.16.1), net.sjrx.intellij.plugins.systemdunitfiles (0.3.2), some.awesome (1.14), com.kite.intellij (1.9.2), ru.meanmail.plugin.requirements (2021.3-211)
```

ERROR TRACE:
```
Details: Current thread: Thread[ApplicationImpl pooled thread 13,4,Idea Thread Group] 201740419
; dispatch thread: false; isDispatchThread(): false
SystemEventQueueThread: Thread[AWT-EventQueue-0,6,Idea Thread Group] 250919519

java.lang.Throwable: Read access is allowed from event dispatch thread or inside read-action only (see com.intellij.openapi.application.Application.runReadAction())
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:179)
	at com.intellij.openapi.application.impl.ApplicationImpl.assertReadAccessAllowed(ApplicationImpl.java:992)
	at com.intellij.psi.impl.file.impl.FileManagerImpl.findDirectory(FileManagerImpl.java:386)
	at com.intellij.psi.impl.PsiManagerImpl.findDirectory(PsiManagerImpl.java:167)
	at com.intellij.javascript.web.context.WebFrameworkContextImplKt.findWebFrameworkInContext(WebFrameworkContextImpl.kt:60)
	at com.intellij.javascript.web.WebFramework$Companion.forContext(WebFramework.kt:52)
	at com.intellij.javascript.web.lang.html.WebFrameworkHtmlLanguageSubstitutor.getLanguage(WebFrameworkHtmlLanguageSubstitutor.kt:12)
	at com.intellij.psi.LanguageSubstitutors.substituteLanguage(LanguageSubstitutors.java:58)
	at com.intellij.lang.LanguageUtil.getLanguageForPsi(LanguageUtil.java:48)
	at com.intellij.lang.LanguageUtil.getLanguageForPsi(LanguageUtil.java:30)
	at net.seesharpsoft.intellij.plugins.csv.CsvHelper.isCsvFile(CsvHelper.java:52)
	at net.seesharpsoft.intellij.plugins.csv.components.CsvFileAttributes.lambda$cleanupAttributeMap$0(CsvFileAttributes.java:66)
	at java.base/java.util.HashMap.forEach(HashMap.java:1336)
	at net.seesharpsoft.intellij.plugins.csv.components.CsvFileAttributes.cleanupAttributeMap(CsvFileAttributes.java:61)
	at net.seesharpsoft.intellij.plugins.csv.CsvPlugin$1.run(CsvPlugin.java:66)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:998)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsync$5(CoreProgressManager.java:497)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:228)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:178)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:688)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:634)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:64)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:165)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:228)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by SeeSharpSoft** [comment]
Thanks for reporting! I am preparing a hotfix...

**Comment by SeeSharpSoft** [comment]
Issue was auto-closed - keeping it open until confirmation. Fix will be released with 2.16.2, previous version already removed from plugin repo.

**Comment by SeeSharpSoft** [comment]
@Mike3285 You reopened the issue - does this error still occur for you with the latest version?
